release/action.yaml

```diff
@@ -28,6 +31,24 @@ runs:
       if: inputs.checkout-repo
       with:
         fetch-depth: 0
-    - run: |
-        echo "Release not implemented yet"
-        exit 1
+    - name: Semantic Release
+      id: release
+      uses: cycjimmy/semantic-release-action@v2
+      with:
+        dry_run: ${{ inputs.dry-run }}
+        extra_plugins: ${{ inputs.extra-plugins }}
+      env:
+        GITHUB_TOKEN: ${{ inputs.github-token }}
+    - name: Fetch Tags
+      if: steps.release.outputs.new-release-published == 'true'
+      shell: bash
+      run: |
+        git fetch --tags
+        git clean -fd
+      env:
+        GITHUB_TOKEN: ${{ inputs.github-token }}
+    - name: Major branch
+      if: steps.release.outputs.new-release-published == 'true'
+      uses: open-turo/action-major-release@v1
+      with:
+        major-version: ${{ steps.release.outputs.new-release-major-version }}
```

Comment:
```suggestion
```
We don't need to publish a major version branch for TF repos - so this can all be deleted.

Reply:
This has been addressed.